seqr/views/apis/report_api.py

```diff
@@ -782,7 +819,7 @@ def _get_participant_row(individual):
         'reported_race': GREGOR_ANCESTRY_MAP.get(individual.population, 'Unknown'),
         'ancestry_detail': GREGOR_ANCESTRY_DETAIL_MAP.get(individual.population),
         'reported_ethnicity': ANCESTRY_MAP[HISPANIC] if individual.population == HISPANIC else 'Unknown',
-        'recontactable': None,  # TODO populate airtable data once new columns are confirmed
+        'recontactable': airtable_sample.get('Recontactable') or 'Unknown',
```

Comment:
no because if the value that comes back from airtable is `{'Recontactable': ''}` then your change would have us report it as blank when the way it is currently written has it report as 'Unknown'